### PR TITLE
Adds the "atheist" trait

### PR DESCRIPTION
**Pull request by monochromatism**
[feature]
## About the PR 

Adds a new trait, "atheist", that affects how you interact with prayer and chaplains. Namely:

- Prayer yields nonthreatening oxygen damage.
- Bible-based healing automatically fails, but the brain damage from this failure is halved for fairness.
- Holy water-based brain damage healing fails as well.
- Player cannot biblefart.

## Why's this needed? 

It's funny and flavorful.

## Changelog

```
(u)monochromatism
(+) Added the "atheist" trait. Disables faith healing and has interesting flavor text about prayer, but might have an upside...
```

### CODE REVIEW

File: code/datums/traits.dm [hunk 1]
```diff
@@ -1017,3 +1017,10 @@ obj/trait/pilot
 	points = 1
 	isPositive = 0
 
+/obj/trait/atheist
+	name = "Atheist (0)"
+	cleanName="Atheist"
```
Comment (ZeWaka) [inline code/datums/traits.dm:1022]:
```suggestion
	cleanName = "Atheist"
```
please be consistent

File: code/modules/chemistry/Reagents-Base.dm [hunk 2]
```diff
@@ -825,8 +825,11 @@ datum
 						M.change_vampire_blood(-burndmg)
 						reacted = 1
 					else if (method == TOUCH)
-						boutput(M, "<span class='notice'>You feel somewhat purified... but mostly just wet.</span>")
-						M.take_brain_damage(-10)
+						if (M?.traitHolder?.hasTrait("atheist"))
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-Base.dm:828]:
```suggestion
						if (M.traitHolder?.hasTrait("atheist"))
```
not needed to check, already done above

File: code/obj/item/storage/bible.dm [hunk 3]
```diff
@@ -83,7 +83,7 @@ var/global/list/bible_contents = list()
 		else if (!isdead(M))
 			var/mob/H = M
 			// ******* Check
-			if ((ishuman(H) && prob(60)))
+			if ((ishuman(H) && prob(60) && !(M?.traitHolder?.hasTrait("atheist"))))
```
Comment (ZeWaka) [inline code/obj/item/storage/bible.dm:86]:
```suggestion
			if ((ishuman(H) && prob(60) && !(M.traitHolder?.hasTrait("atheist"))))
```
already done

File: code/obj/item/storage/bible.dm [hunk 4]
```diff
@@ -94,7 +94,10 @@ var/global/list/bible_contents = list()
 				logTheThing("combat", user, M, "biblically healed [constructTarget(M,"combat")]")
 			else
 				if (ishuman(M) && !istype(M:head, /obj/item/clothing/head/helmet))
-					M.take_brain_damage(10)
+					if (M?.traitHolder?.hasTrait("atheist"))
```
Comment (ZeWaka) [inline code/obj/item/storage/bible.dm:97]:
```suggestion
					if (M.traitHolder?.hasTrait("atheist"))
```
already done

File: code/obj/item/storage/bible.dm [hunk 5]
```diff
@@ -161,9 +164,13 @@ var/global/list/bible_contents = list()
 			user.visible_message("<span class='alert'>[user] farts on the bible.<br><b>The gods seem to approve.</b></span>")
 			return 0
 
-		user.visible_message("<span class='alert'>[user] farts on the bible.<br><b>A mysterious force smites [user]!</b></span>")
-		user.gib()
-		return 0
+		if (user?.traitHolder?.hasTrait("atheist"))
```
Comment (ZeWaka) [inline code/obj/item/storage/bible.dm:167]:
```suggestion
		if (user.traitHolder?.hasTrait("atheist"))
```
already done

File: code/obj/item/storage/bible.dm [hunk 6]
```diff
@@ -161,9 +164,13 @@ var/global/list/bible_contents = list()
...
+			user.visible_message("<span class='alert'>[user] farts on the bible with particular vindication.<br><b>Against all odds, [user] remains unharmed!</b></span>")
+			return
```
Comment (ZeWaka) [inline code/obj/item/storage/bible.dm:169]: return a value

**Review by ZeWaka — CHANGES_REQUESTED** [review]
Please fix your changelog format.

**Comment by Gannets** [comment]
Funny little flavour trait, I'd probably pick this on some characters. 👍

**Comment by mordent-goonstation** [comment]
As I sometimes encourage players to use `pray` to talk to me (as an admin) about some in-round stuff (e.g. if they're running a gimmick and I'm trying to ask them questions about how to spice it up), if they have to resort to adminhelps because they can't use `pray` that sucks.

**Comment by monochromatism** [comment]
Oops. I hadn't really considered that. I haven't had an admin help me with spicing up my gimmicks in months. Maybe allowing atheists to pray after a few failed attempts, along with a message about feeling ridiculous, would be alright then?

**Comment by Tarmunora** [comment]
I think just letting the prayer go through, along with a funny message to the player, would suffice

**Comment by monochromatism** [comment]
Now it tells you you feel silly when praying. Trait description altered accordingly.

**Comment by pali6** [comment]
Maybe it could show the admins something like `PRAYER (atheist): ...` so we can ignore / poke fun at the person appropriately.

**Comment by monochromatism** [comment]
Good idea. Added.

**Review by pali6 — APPROVED** [review]
I wonder if resistance to biblefarts might be too powerful for a trait but we'll see.

**Comment by Tiggersaurus** [comment]
Brown note spammers for one ...

**Review by Tarmunora — APPROVED** [review]
Doesn't look like it affects brown note at all - atheists forcing people to fart on the bible will still gib the atheist (a good thing)

**Pull request merged**